nikola/conf.py.in

```diff
@@ -1030,6 +1030,14 @@ MARKDOWN_EXTENSIONS = ['fenced_code', 'codehilite', 'extra']
 # change it for a FeedBurner feed or something else.
 # RSS_LINK = None
 
+# The following settings allow to disable specific parts of the indexes plugin
```

Comment:
Wall of mumbo jumbo.

> Special settings to disable only parts of the indexes plugin (to allow RSS but no blog indexes, or to allow blog indexes and Atom but no site-wide RSS). Use with care.

Also, those should appear as close to the bottom of the file as possible.